src/BestBooks.js

```diff
@@ -8,21 +12,44 @@ class BestBooks extends React.Component {
     }
   }
 
-  /* TODO: Make a GET request to your API to fetch books for the logged in user  */
+  /* Done: Make a GET request to your API to fetch books for the logged in user  */
+
+  getBooksInfo = async () => {
+    let books = await axios.get('http://localhost:3001/books')
+
+    this.setState({
+      books: books.data
+    })
+  }
+
+  componentDidMount() {
+    this.getBooksInfo();
+  }
 
   render() {
 
     /* TODO: render user's books in a Carousel */
+    let booksToShow = this.state.books.map((book, idx) => (
+      <CarouselItem
+        key={idx}>
+        <h2>{book.title}</h2>
+        <p>{book.status}</p>
+      </CarouselItem>
+    ))
+
```

Comment:
-  I believe we need both `<Carousel />` and `<Carousel.Item />` from bootstrap for this to work 👍  See docs here: [Carousel React Bootstrap](https://react-bootstrap.github.io/components/carousel/)
- Grab TA to verify your Carousel is working properly